app/cypress/e2e/confirm_components_rendered.cy.js

```diff
@@ -0,0 +1,29 @@
+describe('GDS and MOJ components are available', () => {
+    beforeEach(() => {
+        cy.visit('/home')
+        cy.injectAxe()
+    })
+
+    it('displays a GDS summary element', () => {
+        cy.checkA11y()
+
+        cy.get('summary').first()
+            .should('contain.text', 'Help with nationality')
+    })
+
+    it('displays a MOJ password reveal element', () => {
+        cy.checkA11y()
```

Comment:
Be tempted to move this into an after hook or before hook.

Reply:
Makes sense 👍 

Once we have a few tests in place I'd be interested in seeing the performance hit of running this as may be better to selectively run it if it's slowing run times down.